easybuild/easyblocks/n/ncurses.py

```diff
@@ -34,18 +36,21 @@ def configure_step(self):
     def sanity_check_step(self):
         """Custom sanity check for ncurses."""
 
+        libs = ["form", "menu", "ncurses", "panel"]
+
         custom_paths = {
-                        'files' : ['bin/%s' % x for x in ["captoinfo", "clear", "infocmp", "infotocap", "ncurses5-config",
-                                                          "reset", "tabs", "tic", "toe", "tput", "tset"]] +
-                                  ['lib/lib%s.a' % x for x in ["form", "form", "menu", "menu_g", "ncurses", "ncurses++",
-                                                               "ncurses_g", "panel", "panel_g"]],
-                        'dirs' : ['include']
-                       }
+            'files': ['bin/%s' % x for x in ["captoinfo", "clear", "infocmp", "infotocap", "ncurses5-config",
+                                             "reset", "tabs", "tic", "toe", "tput", "tset"]] +
+                     ['lib/lib%s%s.a' % (x, y) for x in libs for y in ['', '_g', 'w', 'w_g']] +
+                     ['lib/lib%s%s.so' % (x, y) for x in libs for y in ['', 'w']] +
+                     ['lib/libncurses++%s.a' % x for x in ['', 'w']],
+            'dirs': ['include', 'include/ncursesw']
```

Comment:
I would even dare to drop point (iii). I can find no reference about that var anywhere. Even google know nothing.
If a specific easyconfigs depends on it, it should happen there. But g++ and icpc don't use it.